src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -40,25 +40,24 @@
     public static final String DESCRIPTION_DESC_TRANSPORT = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_TRANSPORT;
     public static final String PRICE_DESC_CHICKEN = " " + PREFIX_PRICE + VALID_PRICE_CHICKEN;
     public static final String PRICE_DESC_TRANSPORT = " " + PREFIX_PRICE + VALID_PRICE_TRANSPORT;
-    public static final String TAG_DESC_CLAIMABLE = " " + PREFIX_TAG + VALID_TAG_DISCOUNTED;
-    public static final String TAG_DESC_DISCOUNTED = " " + PREFIX_TAG + VALID_TAG_CLAIMABLE;
+    public static final String TAG_DESC_CLAIMABLE = " " + PREFIX_CATEGORY + VALID_TAG_CLAIMABLE;
+    public static final String TAG_DESC_DISCOUNTED = " " + PREFIX_CATEGORY + VALID_TAG_DISCOUNTED;
 
     // '&' not allowed in descriptions
     public static final String INVALID_DESCRIPTION_DESC = " " + PREFIX_DESCRIPTION + "James&";
     // 'a' not allowed in prices
     public static final String INVALID_PRICE_DESC = " " + PREFIX_PRICE + "911a";
     // '*' not allowed in tags
-    public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*";
+    public static final String INVALID_TAG_DESC = " " + PREFIX_CATEGORY + "hubby*";
```

Comment:
Should be renamed as INVALID_CATEGORY_DESC. Same for the Strings in lines 35, 36, 43, 44.